packages/viewer/tests/cypress/tests-e2e/drawing.cy.ts

```diff
@@ -717,12 +723,12 @@ describe('Drawing module tests', () => {
             cy.log('should create a polygon by re-clicking the first point')
             cy.get('[data-cy="ol-map"]').click(100, 250)
 
-            let kmlId = null
+            let kmlId: string | null = null
```

Comment:
when migrating to TS, I would prefer we use `undefined` for non set variables, it's easier to pass down as a param with the optional sign (than to explicitly state it can be null)

So a good rule of thumbs when migrating : change all `null` possible value to `undefined`